### ISSUE TITLE
Up and down arrows/history in command mode

### ISSUE THREAD

**Issue by Kazark** [PolyglotSymposium/void#35]
Make the history feature of command mode work, particularly being able to up and down error to go through past commands.

**Comment by mjgpy3** [comment]
For simplicity's sake, this probably doesn't include partial matching, right?

e.g.

```
:command<CR>
*Other commands not starting with "co"*
```

Then

```
:co
```

Then up arrow key yields

```
:command
```

**Comment by Kazark** [comment]
Resolved in #40
